packages/gatsby-cli/src/reporter/errors.ts

```diff
@@ -3,6 +3,8 @@ import stackTrace from "stack-trace"
 import { prepareStackTrace, ErrorWithCodeFrame } from "./prepare-stack-trace"
 import { isNodeInternalModulePath } from "gatsby-core-utils"
 import { IStructuredStackFrame } from "../structured-errors/types"
+import { readFileSync } from "fs"
```

Comment:
```suggestion
import { readFileSync } from "fs-extra"
```

`fs-extra` doesn't do anything with `readFileSync` but should we still import from `fs-extra` to get improvements from `graceful-fs`?